app/src/main/java/space/taran/arkmemo/data/repositories/ArkFiles.kt

```diff
@@ -0,0 +1,13 @@
+package space.taran.arkmemo.data.repositories
+
+import space.taran.arkmemo.data.repositories.ArkFiles.ARK_FOLDER
+import space.taran.arkmemo.data.repositories.ArkFiles.VERSIONS_FOLDER
+import java.nio.file.Path
+
+object ArkFiles {
+    const val ARK_FOLDER = ".ark"
```

Comment:
(Future work) We need to extract this variable into the Android bindings, it is defined in the library: https://github.com/ARK-Builders/arklib/blob/fea125e8330243522183c187a467dc762dec1fb7/src/lib.rs#L22